README.md

```diff
@@ -46,6 +47,81 @@ npx truffle compile
 npm run networks-inject 
 ```
 
+# Deploying a simple market maker scenario to Rinkeby:
+
+The following script does deploy a simple order market maker order and a relevant
+```
+// Get token id of DAI
+npx truffle exec scripts/stablex/invokeViewFunction.js 'tokenAddressToIdMap' '0x5592EC0cfb4dbc12D3aB100b257153436a1f0FEa' --network rinkeby
+
+//export the result
+export TOKEN_ID_DAI=[Result from last call]
+
+// Get token id of TrueUSD
+npx truffle exec scripts/stablex/invokeViewFunction.js 'tokenAddressToIdMap' '0x0000000000085d4780B73119b644AE5ecd22b376' --network rinkeby
+
+//export the result
+export TOKEN_ID_TUSD=[Result from last call]
```

Comment:
I think we would make the tokenId fetching and exporting in a single line with && to be somewhat faster.

Reply:
That requires parsing this shit output. I don't wanna do it in bash